Helper/Data.php

```diff
@@ -84,28 +102,36 @@ public function getStashApiUrl()
     /**
      * getConfigValue
      * gets config Value for frontend
-     * @param  string $field Field id in system.xml
+     * @param  string $path in config.xml
      * @param  int $storeId StoreID can be null
      * @return string|bool
      */
-    public function getConfigValue($field, $storeId = null)
+    public function getConfigValue($path, $storeID = null, $scopeID = null)
     {
-        return $this->scopeConfig->getValue(
-            $field,
-            ScopeInterface::SCOPE_STORE,
-            $storeId
-        );
+        $valueInStore = $this->config->getValue($path, \Magento\Store\Model\ScopeInterface::SCOPE_STORE);
+        if (isset($valueInStore)) {
```

Comment:
This is the main change (checks `Scope Type`) in this commit.
It checks a lower priority `Scope Type` only if the higher priority `Scope Type`s are all undefined/null.